Application/EdFi.Ods.AdminApi/Features/ClaimSets/ClaimSetModel.cs

```diff
@@ -88,3 +99,90 @@ public class DeleteClaimSetModel : IDeleteClaimSetModel
 
     public int Id { get; set; }
 }
+
+#region Requests
+[SwaggerSchema(Title = "AddClaimSetRequest")]
+public class AddClaimSetRequest
+{
+    [SwaggerSchema(Description = FeatureConstants.ClaimSetNameDescription, Nullable = false)]
+    public string? Name { get; set; }
+
+    [SwaggerSchema(Description = FeatureConstants.ResourceClaimsDescription, Nullable = false)]
+    public List<ResourceClaimModel>? ResourceClaims { get; set; }
+}
+
+[SwaggerSchema(Title = "EditClaimSetRequest")]
+public class EditClaimSetRequest
+{
+    [SwaggerSchema(Description = "ClaimSet id", Nullable = false)]
+    public int Id { get; set; }
+
+    [SwaggerSchema(Description = FeatureConstants.ClaimSetNameDescription, Nullable = false)]
+    public string? Name { get; set; }
+
+    [SwaggerSchema(Description = FeatureConstants.ResourceClaimsDescription, Nullable = false)]
+    public List<ResourceClaimModel>? ResourceClaims { get; set; }
+}
+
+
+public interface IResourceClaimOnClaimSetRequest
+{
+    int ClaimSetId { get; }
+    int ResourceClaimId { get; }
+    public int? ParentResourceClaimId { get; }
+    public ResourceClaimActionModel ResourceClaimActions { get; }
+}
+
+[SwaggerSchema(Title = "AddResourceClaimActionsOnClaimSetRequest")]
+public class AddResourceClaimOnClaimSetRequest : IResourceClaimOnClaimSetRequest
+{
+    [SwaggerExclude]
+    public int ClaimSetId { get; set; }
+
+    [SwaggerSchema(Description = "ResourceClaim id", Nullable = true)]
+    public int ResourceClaimId { get; set; }
+
+    [SwaggerSchema(Description = "Parent ResourceClaim id", Nullable = true)]
+    public int? ParentResourceClaimId { get; set; }
+
+    [SwaggerSchema(Nullable = false)]
+    public ResourceClaimActionModel ResourceClaimActions { get; set; } = new ResourceClaimActionModel();
+}
+
+[SwaggerSchema(Title = "EditResourceClaimActionsOnClaimSetRequest")]
+public class EditResourceClaimOnClaimSetRequest : IResourceClaimOnClaimSetRequest
+{
+    [SwaggerExclude]
+    public int ClaimSetId { get; set; }
+
+    [SwaggerExclude]
+    public int ResourceClaimId { get; set; }
+
+    [SwaggerSchema(Description = "Parent ResourceClaim id", Nullable = true)]
+    public int? ParentResourceClaimId { get; set; }
+
+    [SwaggerSchema(Nullable = false)]
+    public ResourceClaimActionModel ResourceClaimActions { get; set; } = new ResourceClaimActionModel();
+}
+
+
+[SwaggerSchema(Title = "OverrideAuthStategyOnClaimSetRequest")]
+public class OverrideAuthStategyOnClaimSetRequest : FilterResourceClaimOnClaimSet
+{
+    [SwaggerSchema(Description = "Action name", Nullable = false)]
+    public string? ActionName { get; set; }
+
+    [SwaggerSchema(Description = "AuthorizationStrategy name", Nullable = false)]
+    public string? AuthStrategyName { get; set; }
+}
+

```

Comment:
@jagudelo-gap 
OverrideAuthStategyOnClaimSetRequest - this can be directly inherited from "IFilterResourceClaimOnClaimSetModel" - and we do the swagger exclude on "OverrideAuthStategyOnClaimSetRequest" itself for "ClaimSetId" and "ResourceClaimId".